dynamic/Dockerfile

```diff
@@ -1,29 +1,37 @@
-FROM ubuntu:21.04
+# syntax = docker/dockerfile:1.2
 
-ARG DEBIAN_FRONTEND=noninteractive
-ARG TZ=Etc/UTC
+FROM registry.access.redhat.com/ubi8/ubi:latest
```

Comment:
Why use the 'full' `ubi` image here and not the `ubi-minimal` as we do (did) for RHEL?

Reply:
`ubi-minimal` has no `yum`, and I'd much rather use that than `microdnf`.